.github/workflows/update-parsers-pr.yml

```diff
@@ -35,6 +35,11 @@ jobs:
           cp lockfile.json /tmp/lockfile.json
           cat /tmp/lockfile.json | /tmp/jq --sort-keys > lockfile.json
 
+      - name: Commit changes
+        run: |
+          git add lockfile.json
+          git commit -m "Update lockfile.json"
+
```

Comment:
maybe we could also add a cron trigger? Don't know how to formulate it that it only get's executed on master.
```
on:  
  push:
  pull_request:
  schedule:
    - cron: '0 0 * * *'  # every day at midnight
```

Reply:
probably also unrelated to this PR.

Reply:
Yeah I think it's unrelated, and I'd like to keep it like this for now, to synchronise updates with actual activity periods of the plugin.